src/Commands/GenerateArchive.ts

```diff
@@ -125,7 +127,9 @@ export class GenerateArchive implements CommandModule {
       );
       process.exit(0);
     } catch (e) {
-      logger.error(`Error during "archive:generate": ${e.message}`);
+      logger.error(
+        `Error during "archive:generate": ${Helpers.getErrorMessage(e)}`
```

Comment:
What is the reason to use helper in this case? Could it be that there is no error in the `catch` block?